quantile_forest/_quantile_forest.py

```diff
@@ -1357,8 +1357,6 @@ def __init__(
             "max_samples": max_samples,
             "max_samples_leaf": max_samples_leaf,
         }
-        if sklearn_version >= parse_version("1.4.0"):
```

Comment:
This looks good, but please add `"monotonic_cst"` under `"ccp_alpha"` in the `"estimator_params"` list. Unfortunately this omission doesn't currently raise an error in testing, but this ensures that the individual estimators have the attribute (and aligns with how scikit-learn implements this in the forest regressors, e.g. [here](https://github.com/scikit-learn/scikit-learn/blob/54d1ec9b82b14b39cf56aaa7d1d8499fa1d66fe3/sklearn/ensemble/_forest.py#L1896)).